flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
Still :-(

Reply:
Is it possible to check which architecture is currently built?

Reply:
I currently check `DESTCPU`, hopefully that's the "right" way to do things :)